### PR TITLE
#125 Take Alternate Repo New Name from Change Object

### PR DESCRIPTION
**Pull request by criske**
PR for #125

### CODE REVIEW

**Comment by zoeself** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by zoeself** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

File: src/main/java/com/selfxdsd/selfpm/Webhooks.java [hunk 1]
```diff
@@ -143,10 +145,20 @@ public Webhooks(final Self selfCore, final SelfTodos selfTodos) {
...
+                        Provider.Names.GITHUB
                     );
-                    return ResponseEntity.noContent().build();
+                    if(project == null) {
+                        LOG.debug(
+                            "Project " + fullName + " not found either."
+                                + " No Content."
+                        );
+                        return ResponseEntity.noContent().build();
+                    }
```
Comment (amihaiemil) [inline src/main/java/com/selfxdsd/selfpm/Webhooks.java:161]: @criske it should be the other way: first look for ``changes.repository`` and if the full name is not found there, take it from ``repository.full_name``. 

Also please write a TODO for adding a unit test or two for this change :D

**Review by amihaiemil — CHANGES_REQUESTED** [review]

**Comment by amihaiemil** [comment]
@criske looks good, thanks!

**Comment by amihaiemil** [comment]
@rultor merge it please

**Pull request merged**